src/backend/config/configuration.template.yaml

```diff
@@ -37,6 +38,8 @@ tools:
   google_drive:
     client_id:
     client_secret:
+  brave_web_search:
```

Comment:
Let's remove this from config, and just keep it in the secrets file

Reply:
Thank you